conans/test/functional/generators/cmake_find_package_multi_test.py

```diff
@@ -549,6 +549,7 @@ def test_multi_generator_windows(self):
 
     @pytest.mark.skipif(platform.system() != "Darwin", reason="Requires Macos")
     @pytest.mark.tool_xcode
+    @pytest.mark.tool_cmake(version="3.19")
```

Comment:
I am a bit concerned that we are introducing a hard requirement to cmake 3.19, but users will not know that easily, if something fail, specifically because we declare that we support 3.15. 

Is there a way we could introduce the 3.19 requirements so users get the right informative error? Maybe in ``CMakeToolchain`` when ``generator==xcode`` we can introduce a cmake_minimum_required(3.19)?

Reply:
I agree we should introduce some kind of warning. Also to consider, from CMake 3.19 using generator Xcode is only compatible with Xcode > 12.0 but there's an environment variable [CMAKE_XCODE_BUILD_SYSTEM](https://cmake.org/cmake/help/latest/variable/CMAKE_XCODE_BUILD_SYSTEM.html#variable:CMAKE_XCODE_BUILD_SYSTEM) to switch to the XCode 11.X build system, also can be set by [the command line](https://cmake.org/cmake/help/latest/release/3.19.html#generators) (`-T buildsystem=1`).

Reply:
yeah, that seems to require an additional effort from CMakeToolchain side to properly support `CMAKE_XCODE_BUILD_SYSTEM`, but that's probably out of the scope of this PR